packages/atomic/src/components/commerce/sort/option.ts

```diff
@@ -0,0 +1,38 @@
+import {FunctionalComponent} from '@/src/utils/functional-component-utils';
```

Comment:
I don't see a spec file for commerce/sort/option.ts.

I see we have one for common/sort/options.ts though.

Is this an oversight?